support/build/extensions/no-debug-non-zts-20121212/sundown

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+dep_name=$(basename $BASH_SOURCE)
+
+source $(dirname $BASH_SOURCE)/../pecl
```

Comment:
I mean sourcing the original file directly instead of sourcing a copy of the original file, which would itself source files from the submodule.

You said you tried creating a symlink, which indeed does not work as ``$BASH_SOURCE`` would be the symlink in the file, not the real path. But I think sourcing the oroginal file directly from here would work (and avoid the need to maintain our ``pecl`` copy in sync with the original one)